src/components/Quotes.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+
+function Quotes() {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    const send = () => {
+      setLoading(true);
+      try {
+        fetch('https://api.api-ninjas.com/v1/quotes?category=money', {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'tTySwiau1PsXJs5wvhf1dw==NK8bNzv70yBdLNYz' },
+          contentType: 'application/json',
+        })
+          .then((response) => response.json())
+          .then((json) => setData(json));
+      } catch {
+        setHasError(true);
+      }
+      setLoading(false);
+    };
```

Comment:
- Optional
In this case, the `try` and `catch` block is placed around the fetch operation, which is asynchronous and returns a Promise. The catch block won't catch any errors that occur during the execution of the fetch operation because it's outside the Promise chain.
To correctly handle errors in an asynchronous context like this, you should `catch` inside the fetch Promise chain. Like this.
`promise().then().then().catch()`